.travis.yml

```diff
@@ -60,7 +60,7 @@ env:
     - JHI_APP=vuejs-fulli18n-es
     - JHI_APP=vuejs-gateway
     - JHI_APP=vuejs-gradle-session
-    - JHI_APP=vuejs-ws JHI_PROTRACTOR=1
+    - JHI_APP=vuejs-ws-theme JHI_PROTRACTOR=1
```

Comment:
To reduce build time, we can remove this specific new build. 
Then, we can use different theme in existing builds